static/styles/modal.css

```diff
@@ -50,13 +50,34 @@
     font-size: 1.375rem;
     font-weight: 600;
     line-height: 1.25;
+    display: flex;
+    padding-right: 1rem;
 
     /* help_link_widget margin for the fa-circle-o. */
     .help_link_widget {
         margin-left: 5px;
     }
 }
 
+.modal__title div {
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    max-width: 380px;
+}
+
+@media screen and (max-width: 540px) {
+    .modal__title div {
+        max-width: 250px;
+    }
+}
+
+@media screen and (max-width: 389px) {
+    .modal__title div {
+        max-width: 140px;
+    }
+}
+
```

Comment:
Instead of all these hard coded `max-width`s, simply giving the `modal__title` a `max-width` of around 80% should work I think

Reply:
Yes, it works. Thanks!